js/validation.js

```diff
@@ -0,0 +1,19 @@
+let form = document.getElementById('form')
+let email = document.getElementById('mail')
+let error = document.querySelector('.error')
+form.addEventListener('submit', (event)=>{
+    event.preventDefault()
```

Comment:
Well done overall 💯 but the preventDefault() method of the [Event](https://developer.mozilla.org/en-US/docs/Web/API/Event) interface tells the [user agent](https://developer.mozilla.org/en-US/docs/Glossary/User_agent) that if the event does not get explicitly handled ❌, its default action should not be taken as it normally would be. Therefore this line is causing your `form` to not get submitted (the default behavior) even when it's error-free, removing it makes everything works perfectly. 😄